src/http.rs

```diff
@@ -966,6 +967,9 @@ fn vrps(
         }
     };
 
+    let (_start, done, _duration) = origins.update_times();
+    let http_date = StrftimeItems::new("%a, %d %h %Y %H:%M:%S GMT");
```

Comment:
Ideally, this would be a constant with an array of `chrono::format::Item`s. In its current form, the string needs to be parsed for each request which is a bit wasteful since it never changes.

Reply:
I had a look at the CSV date handling, but couldn't find how to format the weekday as string.

Naively (perhaps), I would hope that I can use the format string to create the Items just once, but I didn't get that to work with "const". As I said, no experience with Rust.

Reply:
No worries!

Looks like the week day and month names are called ‘fixed’. So, the week day would be a `Item::Fixed(Fixed::ShortWeekdayName)` (assuming you import the bare names by doing a `use chrono::format::{Item, Fixed}` up top).

The string parsing is just a regular function so it has to happen at runtime. (Rust has something called `const fn` as functions that are evaluated by the compiler but they are somewhat limited in what they can do.)

Reply:
Almost there. Thanks for the help so far. In Java, for instance, I would be able to build a static final DateFormat thingy by parsing the format string once.

Anyway, is some magic needed to ensure that this will always use the ENGLISH locale?

Reply:
If I understand the chrono crate correctly, enabling locales requires adding a feature. So we should be fine.

It might be worthwhile to add a test to make sure?

Reply:
For that I'd need much more hand holding :-)

Reply:
I can also do that later, unless you want to give it a spin?

Reply:
"Later" sounds good to me...